hostManager.js

```diff
@@ -91,6 +92,9 @@ export default class HostManager {
    * @returns {HostEndPoint}
    */
   #createHttpEndPoint(name, options, service) {
+    let externalCommands = LoadCommand.processSync(
+      service.settings._options.Settings.LibPath
+    );
```

Comment:
Command or External command must load in service not in create endpoint! exactly only in `HostService` class.